### ISSUE TITLE
Add a database connection.

### ISSUE THREAD

**Issue by Rihoj** [goFileImporter/file-importer#8]
Possible options include gorm.

**Comment by Rihoj** [comment]
@catdevman, since you have previously worked with getting a connection to a database with golang I thought this would be easy for you to knock out if you want. If you don't want it feel free to pass it back to me.

**Comment by catdevman** [comment]
@Rihoj Yeah I should be able to do that... Hopefully I can add this functionality to the managers so it doesn't have to be figured out for each type of manager but we will see what I can come up with.
